packages/client/components/ActivityLibrary/CreateNewActivity/CreateNewActivity.tsx

```diff
@@ -309,23 +318,40 @@ export const CreateNewActivity = (props: Props) => {
           </div>
           <div className='w-full px-4'>
             <NewMeetingTeamPicker
-              teamsRef={data.viewer.teams}
+              teamsRef={teams}
               selectedTeamRef={selectedTeam}
               onSelectTeam={(teamId) => {
-                const newTeam = data.viewer.teams.find((team) => team.id === teamId)
+                const newTeam = teams.find((team) => team.id === teamId)
                 newTeam && setSelectedTeam(newTeam)
               }}
             />
           </div>
         </div>
         {error && <div className='px-4 text-tomato-500'>{error.message}</div>}
         <div className='mt-auto flex w-full bg-slate-200 p-2 shadow-card-1'>
-          <BaseButton
-            className='mx-auto h-12 rounded-full bg-sky-500 text-lg font-semibold text-white hover:bg-sky-600 focus:outline-none focus:ring-2 focus:ring-sky-600 focus:ring-offset-2 active:ring-sky-600'
-            onClick={createCustomActivityLookup[selectedActivity.type]}
-          >
-            Confirm Format & Team
-          </BaseButton>
+          {selectedTeam.tier === 'starter' ? (
+            <div className='mx-auto flex h-12 items-center gap-24'>
+              <div className='w-96'>
+                Upgrade to the <b>Team Plan</b> to create custom activities unlocking your team’s
```

Comment:
I personally like that it's a bit sales-y, since we're trying to sell users on Parabol's value anytime we're showing the upgrade CTA. In any case, this is easy to change in the future